packages/components/src/components/quickfact/quickfact.stories.ts

```diff
@@ -0,0 +1,249 @@
+  args: overrideArgs([
+    {
+      type: 'slot',
+      name: 'default',
+      value: `<div class="slot slot--border slot--text h-12">Default slot</div>`
+    },
+    {
+      type: 'slot',
+      name: 'summary',
+      value: `<div slot="summary"> <p class="text-base font-normal leading-normal  sm:text-3xl sm:leading-tight">Lorem Ipsum</p>
+            <div class="text-base font-normal leading-normal sm:text-xl">Con sectetur adipiscing elit</div></div>`
+    }
+  ]),
+
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [withActions] as any
+};
+/**
+ * This shows sd-quickfact in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  }
+};
+
+/**
+ * This shows sd-quickfact is displayed differently depending on whether or not the default slot is filled.
+ */
+
+export const OpenXContent = {
+  name: 'Open x Content',
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: {
+          type: 'slot',
+          name: 'default',
+          values: [
+            {
+              title: 'with content',
+              value: `<div class="slot slot--border slot--text h-12">Default slot</div>`
+            },
+            {
+              title: 'without content',
+              value: ''
+            }
+          ]
+        },
+        x: {
+          type: 'attribute',
+          name: 'open',
+          values: [false, true]
+        }
+      },
+      args
+    });
+  }
+};
+
+/**
+ * This shows sd-quickfact in a mobile view. **Please navigate to the `Mobile` story** (you are now on the `Docs` page) to accurately view this behavior.
+ */
+
+export const Mobile = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  },
+  parameters: {
+    viewport: {
+      viewports: INITIAL_VIEWPORTS,
+      defaultViewport: 'iphone6'
+    }
+  }
+};
+
+/**
+ * Use the 'base', 'summary-container', 'content', and 'button' parts to style the sd-quickfact component.
+ */
+export const Parts = {
+  parameters: {
+    controls: {
+      exclude: ['base', 'summary-container', 'content', 'button']
+    }
+  },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: {
+          type: 'template',
+          name: 'sd-quickfact::part(...){outline: solid 2px red}',
+          values: ['base', 'summary-container', 'content', 'button'].map(part => {
+            return {
+              title: part,
+              value: `<style>#part-${part} sd-quickfact::part(${part}){outline: solid 2px red; outline-offset: -2px;}</style><div id="part-${part}">%TEMPLATE%</div>`
+            };
+          })
+        }
+      },
+      args,
+      constants: [
+        {
+          type: 'attribute',
+          name: 'open',
+          value: true
+        }
+      ]
+    });
+  }
+};
+
+/**
+ * Use the `default`, `icon`, `summary`, `expand-icon` and `collapse-icon` slots to add content to the step..
+ */
+
+export const Slots = {
```

Comment:
For the whole story, I have some questions:

1. can we arrange them somehow different to prevent having the vertical layout shift between the first two elements?
2. can we give the `summary` slot a specific width? looks a bit broken like this.
3. When loading the page, the 4th and 5th variant are somehow buggy. The `expand-icon` is obviously shown by the 4th variant but the `collapse-icon` is not shown as the 5th variant is not `open` by default. Therewith you have to click on the item to see the slot and then you don't have an icon to click anymore. Somehow I managed to close the 5th item by clicking on the slot in the 4th item...